view-user/src/components/common/BingoResult/BingoResult.module.css

```
@@ -0,0 +1,53 @@
+.container {
+    /* height: 100vh; */
+    width: 100vw;
+    background-color: #C9AAAA;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .list_container {
+    height: 90vh;
+    width: 90vw;
+  }
+  
+  .card_frame {
+    border-radius: 3px;
+    border-width: 0px;
+    border-style: solid;
+    border-color: #333;
+    padding: 2rem;
+    background-color: #D9D9D9;
+  }
+
+  .card_text {
+    font-size: 3rem;
```

```suggestion
    font-size: 2.5rem;
```
ヘッダーより文字大きいの個人的に気になっただけ
どっちでもいいよ